src/components/EvComponents/SectionWithText.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useTheme, Grid, Container, Box } from "@mui/material";
+import { H2 , Paragraph , H6 , H5 } from "components/EvComponents/Typography";
+import KeyboardDoubleArrowRightOutlinedIcon from '@mui/icons-material/KeyboardDoubleArrowRightOutlined';
+import BazarButton from "components/BazarButton";
+
+const SectionWithText = (props) => {
+    const theme = useTheme();
+    const { title, text , cmd} = props.data;
+  return <Box sx = {{pt : 2}}>
+    <H2
+    color={theme.palette.primary.main}
+    >
+        {title}
+    </H2>
+    <Paragraph
+    color = "grey.600"
+    >
+        {text}
+    </Paragraph>
+    <BazarButton>
+    <KeyboardDoubleArrowRightOutlinedIcon fontSize="inherit"  color = "primary" />
+    <H6
```

Comment:
user H4 or something with bigger font size .. make it bold to be like the original one 
![image](https://user-images.githubusercontent.com/22705909/181508823-b974a614-7f07-4524-b982-16bfb6c2b26e.png)